ROX-17332: Update docs to point to support matrix (#69757)
openshift/openshift-docs: modules/installation-methods-for-different-architectures.adoc

= Installation methods for different architectures

[role="_abstract"]

> {rh-rhacs-first} supports the following architectures. For information on supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix].

Reviewer: I'd suggest to rephrase this somewhat. E.g.

```suggestion
For information on supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix]. Additionally, the following table gives information about installation methods available per architecture.
```

> .Architectures and recommended installation methods

Reviewer: [nit] For ppc64le and s390x there is only one _available_ option, it's not a _recommendation_, at least as it seems to me.

```suggestion
.Supported installation methods per architecture
```

[%autowidth]
|===
|*Supported architectures*|*Supported installation methods*

|AMD64
a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)

| ppc64le ({ibm-power})

> .2+a|Operator (the only supported install method)

Reviewer 2: Beyond scope of this but:

```suggestion
.2+a|Operator (the only supported installation method)
```

Alternatively, I wonder if you could just list "Operator" and leave out the "the only supported installation method" part.
